src/scenes/Partners/Detail/PartnerDetail.tsx

```diff
@@ -133,9 +134,21 @@ const PartnerDataButtons = ({ partner, editPartner }: PartnerViewEditProps) => (
       label="Status"
       onClick={() => editPartner('active')}
       secured={partner?.active}
-      startIcon={<TimelineIcon color="info" />}
+      startIcon={
+        partner?.active.value ? (
+          <TimelineIcon color="info" />
+        ) : (
+          <InactiveStatusIcon color="error" />
+        )
+      }
```

Comment:
For this, I would create a separate component, called `StatusIcon` to avoid making this component big since the file already has multiple components and a bigger component in it.
This will also help if we want to reuse this kind of logic in different places.

Something like this:
```tsx
const StatusIcon = ({ isActive }: { isActive: Maybe<boolean> | null }) =>
  isActive ? (
    <TimelineIcon color="info" />
  ) : (
    <InactiveStatusIcon color="error" />
  );
  ```
  
  And then:
```suggestion
      startIcon={<StatusIcon isActive={partner?.active.value} />}
```

Reply:
@andrewmurraydavid, where in the directory structure should the new StatusIcon component / .tsx file be located? I assume when you say it should be a separate component you are saying create a new file called StatusIcon.tsx where the StatusIcon function will be exported.

Reply:
I added the function to the InactiveStatusIcon.tsx file so that future users can see how those can be used together.  Let me know if you want me to move it somewhere else.

Reply:
For now we can keep it in the `src/scenes/Partners/Detail/PartnerDetail.tsx` file since it's the only place we're using it. When we end up chaning other places then a better place can be found for it.